specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,48 @@
+StartInBackgroundMode
+===
+
+# Background
+By default, WebView2 instances get created with some priorities set higher, assuming the WV2 is being created in the foreground for an important purpose. However, not every developer wants to boost the WebView2 prelaunch phase at the cost of utilizing more system resources. The `StartInBackgroundMode` property specifies that whether WebView2 created with launch boost or not.
+
+
+# Description
+Setting `StartInBackgroundMode` to TRUE will signal WebView2 to start in a 'background mode' which will attempt to use less system resources during the creation and initial load. Default is FALSE.
+
+# Examples
+## Win32 C++
+```cpp
+auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+options->put_StartInBackgroundMode(TRUE));
+HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
+  nullptr, m_userDataFolder.c_str(), options.Get(),
+  Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+     this, &AppWindow::OnCreateEnvironmentCompleted).Get());
+```
+## WinRT and .NET
+```c#
+auto options = new CoreWebView2EnvironmentOptions();
+options.StartInBackgroundMode = true;
+auto environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+```
+
+
+# API Details
+## Win32 C++
+```cpp
+interface ICoreWebView2StagingEnvironmentOptions3 : IUnknown 

```

Comment:
We should use the expected public API names in the API spec so exclude the 'Staging' and derive from the expected parent (ICoreWebView2EnvironmentOptions2?)

Reply:
Yep. Will use ICoreWebView2EnvironmentOptions3 instead.